apps/shopify/src/skuResolvers.js

```diff
@@ -69,25 +76,29 @@ export const fetchCollectionPreviews = async (skus, config) => {
 
 /**
  * Fetches the product previews for the products selected by the user.
- *
- * Note: currently there is no way to cover the edge case where the user
- *       would have more than 250 products selected. In such a case their
- *       selection would be cut off after product no. 250.
  */
 export const fetchProductPreviews = async (skus, config) => {
   if (!skus.length) {
     return [];
   }
-
   const validIds = filterAndDecodeValidIds(skus, 'Product');
   const shopifyClient = await makeShopifyClient(config);
-  const response = await shopifyClient.product.fetchMultiple(validIds);
+
+  const requests = [];
```

Comment:
I believe this should be a `let` since you're mutating the array below.

Reply:
`const` blocks the reassignment of a variable, mutating an array or object is looked at as fine in JS

eg.
```
const numbersArr = [1]
numbersArr.push(2) 
// Success

const failedNumbersArr = [1]
failedNumbersArr = [...failedNumbersArr, 2]
// Uncaught TypeError: Assignment to constant variable
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Reply:
I think this is actually okay `technically` since the array identifier isn't changing, just the contained values which can still mutate.

I had never run into this before, but it looks like this isn't totally a settled issue, and there are opinions on both sides.